cips/cip-pruning-window.md

```diff
@@ -29,18 +29,25 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
 Nodes that prune block data SHOULD store and distribute data in blocks that were created in the last 30 days.
```

Comment:
```suggestion
Nodes that prune block data SHOULD store and distribute data in blocks that were created in the last 30 days + 1 hour worth of seconds (2700000 seconds). The additional 1 hour acts as a buffer to account for clock drift.
```

Reply:
I'm confused. Where does the 2700000 come from?

60 minutes per hour * 60 seconds per minute   = 3600 seconds
30 days * 24 hours per day * 60 minutes per hour * 60 seconds per minute = 2592000 seconds 

2595600 + 3600 = 2599200

Reply:
is this resolved?

Reply:
I think that `2595600` needs to change to `2599200`

Reply:
Updated suggestion above to reflect `2599200`

Reply:
> I don't understand where any of the existing second numbers come from (2599200 seconds).
> 
> 30 days = 60 * 60 * 24 * 30 = 2592000
> 30 days and 1 hour = 2592000 + 60 * 60 = 2595600
> 
> Am I going mad?

Nope. Since 30 days + 1 hour = 2595600, this should change to **2595600**